cfdm/docstring/docstring.py

```diff
@@ -220,6 +220,39 @@
                 ``3``/``'DETAIL'``) for increasing verbosity, the more
                 description that is printed to convey information
                 about the operation.""",
+    "{{d: `int`}}": """d: `int`
+                The position in the tie points array of the subsampled
+                dimension being interpolated.""",
+    # d1
+    "{{d1: `int`}}": """d1: `int`
+                The position of subsampled dimension 1 (in the sense
+                of CF Appendix J Figure J.2) in the full tie points
+                array.""",
+    # d2
+    "{{d2: `int`}}": """d2: `int`
+                The position of subsampled dimension 2 (in the sense
+                of CF Appendix J Figure J.2) in the full tie points
```

Comment:
There are a few more appendix J figure references here, so please see the previous comment relating to those.

Reply:
Fixed in CF-1.10